t/lib/OpenQA/Test/Utils.pm

```diff
@@ -114,6 +114,14 @@ sub fake_asset_server {
                     $c->rendered(200);
                 }
             }
+
```

Comment:
separating the elsif with a blank line is unusual. Care to delete the blank line?

Reply:
Actually, i commonly separate `elsif` if the blocks are bigger like in this case. And i think it looks better here (in context with the other `if`/`elsif` blocks).